app/src/DataFileGenerator/EventFileGenerator.php

```diff
@@ -51,6 +51,12 @@ public function getEventFromMeetup(int $meetupId)
             $event->setTitle($eventRaw->name);
         }
 
+        if (!empty($eventRaw->venue->name)) {
```

Comment:
I just moved this to be before we look at the full description which is where the guessing really starts. Purely OCD.